grid/Grid-Recon.md

```diff
@@ -0,0 +1,39 @@
+
+## Grid Reconnaisance
+This is mainly to discover the different services, ports, technologies the infrastructure uses.
+This helps understand the underlying technologies, ports and services so we can search for the versions which are vulnerable, try to abuse `wrongly configured settings, try to gain control, escalade, bypass, fuzz` etc.
+Tools used :
+- `kali OS` with pre-installed tools. But you may need to manually install other custom tools like `ffuf` etc.
+- Nmap
+- Nikto
+- Dirb
+#### 1. Nmap : Used for network scanning, port discovery, services using NSE scripts and many more.
+- `5 ports` opened, `5000`, `5001`, `5002`, `5003`, `7000`.
+-  Ports `5000-5003` and `7000` uses `Ajenti http control panel` which is an open source project maintained by : http://ajenti.org/
+- http-methods supported  `GET` , `OPTIONS`, `HEAD`. 
+- Command used : `nmap -p0- -v -A -T4 <IP>` e.g nmap -p0- -v -A -T4 192.168.0.3
+- Output screenshots.
+- Open ports
+![](https://i.imgur.com/6IpWrCj.png)
+- Services
+![](https://i.imgur.com/QbpDcgq.png)
+
+#### 2. Nikto : Used for web server scanning.
+- For both `Network` and `Wrokers node`,
+- `anti-clickjacking X-Frame-Options header` is not present. This  means that this website could be at risk of a clickjacking attack. The X-Frame-Options HTTP response header can be used to indicate whether or not a browser should be allowed to render a page inside a frame or iframe. 
+- `X-XSS-Protection header` is not defined which can lead to `XSS` attacks.
+- `X-Content-Type-Options header` is not set. There is a possibility to `execute style sheet` and `steal` content from another site through content type doesn’t match. More details[ here](https://geekflare.com/secure-mime-types-in-apache-nginx-with-x-content-type-options/).
+- Network
+![](https://i.imgur.com/BeElbwg.png)
+- Alice similar to the other workers like bob etc.
+![](https://i.imgur.com/tGZfM2C.png)
+- Command `nikto -url <url> -C all` , you can also use `nikto -h <url> -C all`. thet are similar.
+#### 3. Dirb : use for directory discovery
+- For the moment no sensitive directories found.
+
```

Comment:
Looks great, Baye.
Could you add a `### Summary` section which summarises your findings in a bullet point with a rough severity level (High, Medium, Low).